aws/validators.go

```diff
@@ -1992,3 +1992,30 @@ func validateCloudFrontPublicKeyNamePrefix(v interface{}, k string) (ws []string
 	}
 	return
 }
+
+func validateSecretManagerSecretName(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
I think you meant to include `ValidateFunc: validateSecretManagerSecretName` in the schema. 😉 Same with the below validation function.

Reply:
yes, I ll add now 👍